backend/futourism/src/main/java/com/bunchofhacks/futourism/controller/POIController.java

```diff
@@ -8,13 +8,17 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.ApplicationContext;
 import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.ArrayList;
 import java.util.List;
 
 //TODO: Add endpoint to dynamically respond to client based on user preferences
+@CrossOrigin(value = { "*" },
+  allowedHeaders = { "GET, POST" }
+)
```

Comment:
Does this work? I tried exactly that and it didn't work

Reply:
This is the code currently running, so I guess yes. It works when you add the `Origin` header.